client/asset/btc/rpcclient.go

```diff
@@ -543,6 +596,37 @@ func (wc *rpcClient) refundAddress() (btcutil.Address, error) {
 	return wc.externalAddress()
 }
 
+type ZecAddressForAccountResult struct {
+	Account          uint32   `json:"account"`
+	DiversifierIndex uint32   `json:"diversifier_index"`
+	ReceiverTypes    []string `json:"receiver_types"`
+	Address          string   `json:"address"`
+}
+
+type ZecUnifiedReceivers struct {
```

Comment:
unexport and/or doc?

Reply:
Will unexport for now, but I suspect that I will need these exported for work with shielded pools.